srv/salt/ceph/openstack/cinder-backup/pool/default.sls

```diff
@@ -0,0 +1,11 @@
+{% set prefix = pillar['openstack_prefix'] + "-" if 'openstack_prefix' in pillar else "" %}
```

Comment:
same comment as in https://github.com/SUSE/DeepSea/pull/1025/files#r196747950..

exemplary for all other occurrences